CMakeLists.txt

```diff
@@ -39,6 +39,12 @@ common_find_package(Qt5Widgets REQUIRED)
 common_find_package(Servus)
 common_find_package_post()
 
+if(Qt5Quick_VERSION VERSION_LESS 5.5)
```

Comment:
maybe the option should not even be here for Qt < 5.5, since we know that it does not work.